cmd/walrus/main.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 var (
-	// to be supplied at build time
+	// to be supplied at build time.
```

Comment:
my (admittedly idiosyncratic) commenting style is "lowercase without period for up to 1 sentence, proper grammar for anything longer." Not going to change that to suit some nonstandard linting option.